### PR TITLE
feat(pagination):create

### PR DESCRIPTION
**Pull request by ArthusLiang**
create functions

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/pagination/pagination-arrow.module.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+@NgModule({
+  declarations: [PaginationArrowComponent],
+  imports: [
+    CommonModule
+  ],
+  exports: [PaginationArrowComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA]
+})
+export class PaginationArrowModule { }
```
Comment (daanishnasir) [inline angular/src/lib/pagination/pagination-arrow.module.ts]: Need to add forRoot to new components, here's an example; it should be in all the component modules

export class TopbarModule {
  static forRoot(): ModuleWithProviders {
    return {
      ngModule: TopbarModule,
      providers: [],
    };
  }
}

File: angular/src/lib/pagination/pagination.component.ts [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        s.selectfocus();
+        ifStop = true;
+        break;
+      default:
+        break;
+    }
+    if (ifStop) {
+      event.stopPropagation();
+    }
+  }
```
Comment (daanishnasir) [inline angular/src/lib/pagination/pagination.component.ts]: lets try to include case's for the angular cdk keycodes as well when handling keydowns

`import {
  DOWN_ARROW,
  END,
  ENTER,
  HOME,
  LEFT_ARROW,
  PAGE_DOWN,
  PAGE_UP,
  RIGHT_ARROW,
  SPACE,
  UP_ARROW,
} from '@angular/cdk/keycodes';`

File: angular/src/lib/pagination/pagination-arrow.module.ts (hunk 1 above)
Comment (ArthusLiang) [reply angular/src/lib/pagination/pagination-arrow.module.ts]: Thanks for you suggestion

File: angular/src/lib/pagination/pagination.component.ts (hunk 2 above)
Comment (ArthusLiang) [reply angular/src/lib/pagination/pagination.component.ts]: updated

**Pull request merged**